classes/CartRule.php

```diff
@@ -565,16 +565,16 @@ public function checkValidity(Context $context, $alreadyInCart = false, $display
         }
 
         if (!$this->active) {
-            return (!$display_error) ? false : Tools::displayError('This voucher is disabled');
+            return (!$display_error) ? false : $this->trans('This voucher is disabled', array(), 'Admin.Orderscustomers.Notification');
```

Comment:
I'm so sorry but I think I made a mistake for this file, it seems it's all front messages, so the domain should be Shop.Notifications.Error

"oups"